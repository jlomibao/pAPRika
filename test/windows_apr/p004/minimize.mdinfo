

   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.1435E+02     4.8935E-01     1.6136E+00     H26        87

 BOND    =        1.1724  ANGLE   =      107.9472  DIHED      =       52.6005
 VDWAALS =      -55.4535  EEL     =     1330.5892  EGB        =     -127.8269
 1-4 VDW =        5.1353  1-4 EEL =    -2128.5535  RESTRAINT  =        0.0391
 EAMBER  =     -814.3893
 NMR restraints: Bond =    0.002   Angle =     0.028   Torsion =     0.009
===============================================================================
