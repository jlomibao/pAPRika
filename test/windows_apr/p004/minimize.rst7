defa
  125
  -0.5274117   4.9674689   1.7866499   1.0104276   4.9063169   1.7850138
  -0.8896826   4.2560169   0.5691146  -0.8907299   4.2909444   3.0254300
  -0.9080732   5.9952428   1.7718220   1.4715392   5.9005566   1.7675292
   1.3129300   4.1661700   0.5682370   1.3216622   4.2049677   3.0237519
   2.6764995   4.0723481   0.0678080   2.6942793   4.1166294   3.5168187
   0.2033209   3.9200010   3.7647557   0.1956542   3.8266071  -0.1472564
   0.1882853   3.4920233   4.9044273   0.1743153   3.3040402  -1.2485650
   2.6488575   4.0208058  -1.0267360   3.2299627   4.9628663   0.3873348
   2.6713045   4.0883907   4.6123180   3.2427341   5.0038074   3.1798411
  -4.6689683   1.9945761   1.7967945  -3.8226356   3.2805178   1.7939571
  -4.2379174   1.3097647   0.5860373  -4.2845367   1.3419547   3.0411414
  -5.7457485   2.1981376   1.7733597  -4.4361240   4.1886860   1.7765069
  -3.0337663   3.1559586   0.5761546  -3.0605718   3.1854772   3.0332598
  -2.2560282   4.2764002   0.0682578  -2.2651064   4.3097170   3.5217801
  -3.3979249   2.0838882   3.7772880  -3.3119776   2.0177567  -0.1326959
  -3.0382167   1.8478191   4.9160230  -2.8697440   1.7242043  -1.2302472
  -2.2324854   4.2233685  -1.0263109  -2.7337446   5.2095575   0.3883298
  -2.2419967   4.2720538   4.6169928  -2.7408248   5.2407265   3.1926032
  -4.1008220  -3.0754955   1.8621183  -4.8312235  -1.7208725   1.8377067
  -3.2962652  -3.0418002   0.6485157  -3.3427094  -3.0259467   3.1066097
  -4.7915458  -3.9265474   1.8548172  -5.9215119  -1.8303782   1.8108417
  -4.3363548  -1.0980586   0.6177330  -4.3980129  -1.0830546   3.0742681
  -4.9337133   0.1258317   0.1030068  -4.9981476   0.1665352   3.5327346
  -3.5851465  -1.8868858   3.8310603  -3.4716144  -1.8968370  -0.0820973
  -3.2086379  -1.6615838   4.9674977  -3.0019415  -1.6627339  -1.1825328
  -4.8852695   0.1095029  -0.9919058  -5.9781804   0.1730501   0.4320972
  -4.9843956   0.1807006   4.6286608  -6.0323228   0.2097921   3.1721075
   0.5806451  -5.0968987   1.9002144  -0.9572889  -5.0346464   1.8964694
   0.9435081  -4.4212050   0.6625816   0.9443524  -4.3830326   3.1176417
   0.9610907  -6.1247419   1.9157602  -1.4196689  -6.0283789   1.9084045
  -1.2587080  -4.3300913   0.6584538  -1.2676833  -4.2954085   3.1134041
  -2.6234242  -4.2372786   0.1611441  -2.6379181  -4.2012700   3.6113852
  -0.1489338  -3.9823767   3.8417935  -0.1411698  -4.0112026  -0.0660374
  -0.1330783  -3.5100967   4.9634205  -0.1190968  -3.5176590  -1.1806628
  -2.5993571  -4.2076937  -0.9342976  -3.1813090  -5.1177385   0.5002557
  -2.6097341  -4.1493505   4.7058960  -3.1854161  -5.0968828   3.2958455
   4.7194329  -2.1211079   1.8481374   3.8747692  -3.4074957   1.8737388
   4.2846634  -1.4619555   0.6243553   4.3381868  -1.4438522   3.0806609
   5.7962997  -2.3243469   1.8259131   4.4895109  -4.3149837   1.8732103
   3.0821457  -3.3094306   0.6558479   3.1173886  -3.2876545   3.1139680
   2.3123248  -4.4448397   0.1687158   2.3157180  -4.3981825   3.6214430
   3.4545484  -2.1710373   3.8357651   3.3579921  -2.1861094  -0.0772933
   3.0933314  -1.9104424   4.9696273   2.9145134  -1.9170693  -1.1806145
   2.2935703  -4.4155242  -0.9268370   2.7931951  -5.3684941   0.5108681
   2.2872347  -4.3380355   4.7155247   2.7891903  -5.3375389   3.3134250
   4.1580111   2.9501918   1.7924205   4.8855986   1.5933356   1.7997810
   3.3592723   2.8915934   0.5761470   3.3946086   2.9285366   3.0343915
   4.8505805   3.7994743   1.7698593   5.9762861   1.6994117   1.7754790
   4.3942162   0.9455966   0.5915017   4.4449939   0.9816757   3.0465204
   4.9854976  -0.2943346   0.1095496   5.0476057  -0.2533728   3.5404394
   3.6299091   1.8027705   3.7810862   3.5328304   1.7301114  -0.1282517
   3.2508767   1.6022078   4.9203422   3.0637819   1.4717313  -1.2235008
   4.9365908  -0.3069372  -0.9853917   6.0299419  -0.3373056   0.4392937
   5.0320875  -0.2375428   4.6363192   6.0825076  -0.3040239   3.1828727
   0.0119650  -0.1270369   4.2392646  -0.9130581  -0.6795230   4.0336590
   0.8599873  -0.7901941   4.0288755   0.0625241   0.7163745   3.5397703
   0.0473356   0.3815779   5.6983847   0.9628384   0.9639751   5.8591544
  -0.0158174  -0.7545687   6.7400949  -0.7922409   1.0674677   5.8652502
  -0.9323450  -1.3391770   6.5938425   0.8307241  -1.4365899   6.5938302
   0.0141693  -0.2145952   8.1837601  -0.8402572   0.4472012   8.3713084
  -0.0306719  -1.0409637   8.9036879   0.9356202   0.3503634   8.3706880
   0.0016852  -0.0009605  -6.0017927  -0.0012133   0.0005805  -9.0000690
   0.0000701   2.2000034 -11.2000238
