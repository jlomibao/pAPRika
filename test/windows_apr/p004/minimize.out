
          -------------------------------------------------------
          Amber 18 SANDER                              2018
          -------------------------------------------------------

| Run on 04/10/2019 at 18:19:07

|   Executable path: sander
| Working directory: /home/jlomibao/pAPRika_fork/pAPRika/test/windows_apr/p004
|          Hostname: entropy.ucsd.edu

  [-O]verwriting output

File Assignments:
|  MDIN: minimize.in                                                           
| MDOUT: minimize.out                                                          
|INPCRD: cb6-but-dum.rst7                                                      
|  PARM: cb6-but-dum.prmtop                                                    
|RESTRT: minimize.rst7                                                         
|  REFC: cb6-but-dum.rst7                                                      
| MDVEL: mdvel                                                                 
| MDFRC: mdfrc                                                                 
|  MDEN: minimize.mden                                                         
| MDCRD: minimize.nc                                                           
|MDINFO: minimize.mdinfo                                                       
|  MTMD: mtmd                                                                  
|INPDIP: inpdip                                                                
|RSTDIP: rstdip                                                                
|INPTRA: inptraj                                                               


 Here is the input file:

PBC MD Simulation                                                              
 &cntrl                                                                        
  imin =          1,                                                           
  ntx =           1,                                                           
  irest =         0,                                                           
  maxcyc =        5000,                                                        
  ncyc =          5000,                                                        
  dt =            0.002,                                                       
  nstlim =        5000,                                                        
  ntpr =          500,                                                         
  ntwe =          500,                                                         
  ntwr =          5000,                                                        
  ntwx =          500,                                                         
  ntxo =          1,                                                           
  ioutfm =        1,                                                           
  ntf =           2,                                                           
  ntc =           2,                                                           
  cut =           999.0,                                                       
  igb =           1,                                                           
  tempi =         298.15,                                                      
  temp0 =         298.15,                                                      
  ntt =           3,                                                           
  gamma_ln =      1.0,                                                         
  ig =            -1,                                                          
  ntp =           0,                                                           
  barostat =      2,                                                           
  ntr =           1,                                                           
  restraint_wt =  50.0,                                                        
  restraintmask = '@DUM',                                                      
  nmropt =        1,                                                           
  pencut =        -1,                                                          
  ntb =           0,                                                           
 /                                                                             
 &wt type = 'END', /                                                           
DISANG = rest.in                                                               
LISTOUT = POUT                                                                 
                                                                               

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

| Flags:                                                                        
| New format PARM file being parsed.
| Version =    1.000 Date = 01/16/19 Time = 17:42:13
 NATOM  =     125 NTYPES =       7 NBONH =      46 MBONA  =      93
 NTHETH =     118 MTHETA =     158 NPHIH =     200 MPHIA  =     375
 NHPARM =       0 NPARM  =       0 NNB   =     836 NRES   =       5
 NBONA  =      93 NTHETA =     158 NPHIA =     375 NUMBND =       6
 NUMANG =      12 NPTRA  =      13 NATYP =       7 NPHB   =       0
 IFBOX  =       0 NMXRS  =     108 IFCAP =       0 NEXTRA =       0
 NCOPY  =       0

 Implicit solvent radii are modified Bondi radii (mbondi)                                                   

|     Memory Use     Allocated
|     Real               14705
|     Hollerith            382
|     Integer            27997
|     Max Pairs              1
|     nblistReal             0
|     nblist Int             0
|       Total              225 kbytes

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| Duplicated    0 dihedrals
| Duplicated    0 dihedrals
Note: ig = -1. Setting random seed to   159989 based on wallclock time in microseconds.

--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

default_name                                                                    

General flags:
     imin    =       1, nmropt  =       1

Nature and format of input:
     ntx     =       1, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       1, ntpr    =     500, ntrx    =       1, ntwr    =    5000
     iwrap   =       0, ntwx    =     500, ntwv    =       0, ntwe    =     500
     ioutfm  =       1, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       2, ntb     =       0, igb     =       1, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     = 999.00000, intdiel =   1.00000
     saltcon =   0.00000, offset  =   0.09000, gbalpha=    1.00000
     gbbeta  =   0.00000, gbgamma =   0.00000, surften =   0.00500
     rdt     =   0.00000, rgbmax  =  25.00000  extdiel =  78.50000
     alpb  =        0

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       1
     restraint_wt =  50.00000

Energy minimization:
     maxcyc  =    5000, ncyc    =    5000, ntmin   =       1
     dx0     =   0.01000, drms    =   0.00010

SHAKE:
     ntc     =       2, jfastw  =       0
     tol     =   0.00001

NMR refinement options:
     iscale  =       0, noeskp  =       1, ipnlty  =       1, mxsub   =       1
     scalm   = 100.00000, pencut  =  -1.00000, tausw   =   0.10000
|  INFO: Old style inpcrd file read


    LOADING THE CONSTRAINED ATOMS AS GROUPS


   5.  REFERENCE ATOM COORDINATES

      
     Mask @DUM; matches     3 atoms

--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

                                                                                
 begin time read from input coords =     0.000 ps



           Begin reading energy term weight changes/NMR restraints
 WEIGHT CHANGES:
                         ** No weight changes given **

 RESTRAINTS:
 Requested file redirections:
  DISANG    = rest.in
  LISTOUT   = POUT
 Restraints will be read from file: rest.in
Here are comments from the DISANG input file:

                       Number of restraints read =     9

                  Done reading weight changes/NMR restraints


 Number of triangulated 3-point waters found:        0

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
      1      -7.9737E+02     1.0105E+01     2.5139E+01     C34       101

 BOND    =       11.0328  ANGLE   =      109.3438  DIHED      =       52.9822
 VDWAALS =      -52.4842  EEL     =     1326.8251  EGB        =     -123.4121
 1-4 VDW =        5.8710  1-4 EEL =    -2127.5276  RESTRAINT  =        0.0000
 EAMBER  =     -797.3691
 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    500      -8.1410E+02     5.0170E-01     1.7558E+00     H3        114

 BOND    =        1.1707  ANGLE   =      107.9986  DIHED      =       52.7462
 VDWAALS =      -55.2505  EEL     =     1329.8963  EGB        =     -127.6655
 1-4 VDW =        5.1269  1-4 EEL =    -2128.1376  RESTRAINT  =        0.0181
 EAMBER  =     -814.1148
 NMR restraints: Bond =    0.002   Angle =     0.015   Torsion =     0.001
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1000      -8.1423E+02     4.9556E-01     1.6181E+00     H8         33

 BOND    =        1.1751  ANGLE   =      107.9724  DIHED      =       52.6675
 VDWAALS =      -55.3519  EEL     =     1330.3066  EGB        =     -127.7708
 1-4 VDW =        5.1364  1-4 EEL =    -2128.3927  RESTRAINT  =        0.0233
 EAMBER  =     -814.2574
 NMR restraints: Bond =    0.002   Angle =     0.018   Torsion =     0.003
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1500      -8.1430E+02     4.9260E-01     1.6165E+00     H8         33

 BOND    =        1.1731  ANGLE   =      107.9577  DIHED      =       52.6310
 VDWAALS =      -55.3965  EEL     =     1330.4552  EGB        =     -127.8050
 1-4 VDW =        5.1372  1-4 EEL =    -2128.4787  RESTRAINT  =        0.0285
 EAMBER  =     -814.3260
 NMR restraints: Bond =    0.002   Angle =     0.022   Torsion =     0.005
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2000      -8.1433E+02     4.9274E-01     1.6153E+00     H8         33

 BOND    =        1.1787  ANGLE   =      107.9511  DIHED      =       52.6132
 VDWAALS =      -55.4214  EEL     =     1330.5426  EGB        =     -127.8352
 1-4 VDW =        5.1356  1-4 EEL =    -2128.5241  RESTRAINT  =        0.0328
 EAMBER  =     -814.3594
 NMR restraints: Bond =    0.002   Angle =     0.024   Torsion =     0.006
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2500      -8.1434E+02     4.9018E-01     1.6145E+00     H8         33

 BOND    =        1.1730  ANGLE   =      107.9485  DIHED      =       52.6049
 VDWAALS =      -55.4361  EEL     =     1330.5673  EGB        =     -127.8287
 1-4 VDW =        5.1362  1-4 EEL =    -2128.5410  RESTRAINT  =        0.0357
 EAMBER  =     -814.3758
 NMR restraints: Bond =    0.002   Angle =     0.026   Torsion =     0.007
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   3000      -8.1435E+02     4.8967E-01     1.6139E+00     H8         33

 BOND    =        1.1716  ANGLE   =      107.9474  DIHED      =       52.6014
 VDWAALS =      -55.4460  EEL     =     1330.5815  EGB        =     -127.8268
 1-4 VDW =        5.1360  1-4 EEL =    -2128.5495  RESTRAINT  =        0.0378
 EAMBER  =     -814.3845
 NMR restraints: Bond =    0.002   Angle =     0.027   Torsion =     0.008
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   3500      -8.1435E+02     4.8940E-01     1.6135E+00     H26        87

 BOND    =        1.1724  ANGLE   =      107.9472  DIHED      =       52.6005
 VDWAALS =      -55.4522  EEL     =     1330.5885  EGB        =     -127.8273
 1-4 VDW =        5.1354  1-4 EEL =    -2128.5529  RESTRAINT  =        0.0389
 EAMBER  =     -814.3885
 NMR restraints: Bond =    0.002   Angle =     0.027   Torsion =     0.009
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   4000      -8.1435E+02     4.8935E-01     1.6136E+00     H26        87

 BOND    =        1.1724  ANGLE   =      107.9472  DIHED      =       52.6005
 VDWAALS =      -55.4535  EEL     =     1330.5891  EGB        =     -127.8269
 1-4 VDW =        5.1354  1-4 EEL =    -2128.5534  RESTRAINT  =        0.0391
 EAMBER  =     -814.3893
 NMR restraints: Bond =    0.002   Angle =     0.028   Torsion =     0.009
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   4500      -8.1435E+02     4.8935E-01     1.6136E+00     H26        87

 BOND    =        1.1724  ANGLE   =      107.9472  DIHED      =       52.6005
 VDWAALS =      -55.4535  EEL     =     1330.5891  EGB        =     -127.8269
 1-4 VDW =        5.1353  1-4 EEL =    -2128.5534  RESTRAINT  =        0.0391
 EAMBER  =     -814.3893
 NMR restraints: Bond =    0.002   Angle =     0.028   Torsion =     0.009
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.1435E+02     4.8935E-01     1.6136E+00     H26        87

 BOND    =        1.1724  ANGLE   =      107.9472  DIHED      =       52.6005
 VDWAALS =      -55.4535  EEL     =     1330.5892  EGB        =     -127.8269
 1-4 VDW =        5.1353  1-4 EEL =    -2128.5535  RESTRAINT  =        0.0391
 EAMBER  =     -814.3893
 NMR restraints: Bond =    0.002   Angle =     0.028   Torsion =     0.009
===============================================================================


  Maximum number of minimization cycles reached.


                    FINAL RESULTS



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.1435E+02     4.8935E-01     1.6136E+00     H26        87

 BOND    =        1.1724  ANGLE   =      107.9472  DIHED      =       52.6005
 VDWAALS =      -55.4535  EEL     =     1330.5892  EGB        =     -127.8269
 1-4 VDW =        5.1353  1-4 EEL =    -2128.5535  RESTRAINT  =        0.0391
 EAMBER  =     -814.3893
 NMR restraints: Bond =    0.002   Angle =     0.028   Torsion =     0.009
===============================================================================
 ------------------------------------------------------------------------------


 Final Restraint Analysis for coords: minimize.rst7                           


 Restraints, deviations, and energy contributions:    pencut =   -1.00

 ------------------------------------------------------------------------------
     First atom        Last atom    curr. value target deviation  penalty
 ------------------------------------------------------------------------------
  DUM  DM1    3 --  C7   CB6    1:    9.283    9.302    0.019    0.002 d    0: 0
  DUM  DM2    4 --  C7   CB6    1:  147.060  147.398    0.338    0.003 a
  DUM  DM2    4 --  DUM  DM1    3:   49.479   49.483    0.004    0.000 t
  DUM  DM1    3 --  C31  CB6    1:   61.891   62.003    0.112    0.000 a
  DUM  DM1    3 --  C7   CB6    1: -145.891 -146.178    0.287    0.003 t
  C7   CB6    1 --  C31  CB6    1:  -73.014  -72.548    0.466    0.007 t
  DUM  DM1    3 --  C    BUT    2:   10.242   10.235    0.007    0.000 d    0: 0
  DUM  DM2    4 --  C    BUT    2:  179.324  180.000    0.676    0.014 a
  DUM  DM1    3 --  C3   BUT    2:  179.433  180.000    0.567    0.010 a
                                       Total distance penalty:      0.002
                                       Total angle    penalty:      0.028
                                       Total torsion  penalty:      0.009
|                               RMS deviation from ideal bonds :      0.0095
|                               RMS deviation from ideal angles:      4.475
 ------------------------------------------------------------------------------

--------------------------------------------------------------------------------
   5.  TIMINGS
--------------------------------------------------------------------------------

|                Calc gb radii              2.21 (21.88% of Gen B)
|                Calc gb diag               3.96 (39.16% of Gen B)
|                Calc gb off-diag           3.94 (38.93% of Gen B)
|                Other                      0.00 ( 0.02% of Gen B)
|             Gen Born time             10.11 (100.0% of Nonbo)
|          Nonbond force             10.11 (90.45% of Force)
|          Bond/Angle/Dihedral        1.03 ( 9.26% of Force)
|          Other                      0.03 ( 0.29% of Force)
|       Force time                11.17 (100.0% of Runmd)
|    Runmd Time                11.17 (99.74% of Total)
|    Other                      0.03 ( 0.25% of Total)
| Total time                11.20 (100.0% of ALL  )

| Number of list builds   :          0

| Highest rstack allocated:          0
| Highest istack allocated:          0
|           Job began  at 18:19:07.156  on 04/10/2019
|           Setup done at 18:19:07.161  on 04/10/2019
|           Run   done at 18:19:18.360  on 04/10/2019
|     wallclock() was called   90028 times

|3D-RISM memory allocation summary
|Type          Maximum        Current   
|Integer       0.00000 GB     0.00000 GB
|Real          0.00000 GB     0.00000 GB
|Logical       0.00000 GB     0.00000 GB
|Character     0.00000 GB     0.00000 GB
|---------------------------------------
|Total         0.00000 GB     0.00000 GB
